kustomize/resource_kustomization_test.go

```diff
@@ -556,8 +663,6 @@ resource "kustomization_resource" "namespacedco" {
 `
 }
 
-//
```

Comment:
These are removed by`go fmt` when I save (either in vscode or vim) - it's not a deliberate change on my part but might be easier to keep the change to save fighting the tooling.